### DISCUSSION TITLE
Three features - Tamil, Import and Sync

### DISCUSSION THREAD

**Discussion by mangoose20026** [ezra-bible-app/ezra-bible-app#1081] [Ideas]
1. Add support as well as a new page for tamil bible like English (https://bolls.life/) to search and read online. Tamil bible is available in all formats whichever you like to use (https://github.com/godlytalias/Bible-Database/tree/master/Tamil)
2. Add support to import sermon text files to search like how we search the bible. That would be really helpful. (For example, import spurgeon sermon text files, import John Bunyan sermons files, import Calvin Sermon files)
3. Add support to Sync sermon text files from website folder link or shared folder link. That would be really helpful. (For example, spurgeon sermon text files placed in a website in a folder, John Bunyan sermons files placed in a website in a folder, import Calvin Sermon files placed in a website in a folder)

**Comment by tobias-klein** [comment]
Hi @mangoose20026
Thanks for your interest in Ezra Bible App! I moved your requests to the discussion section and we can discuss here.

1. *Support for Tamil Bible*. Your request is for searching and reading online. However, this is app is actually an offline app. There are a few Tamil Bible translations available through the ebible.org repository (see below). If you would like to study a Tamil Bible translation in Ezra that is not available yet, it would have to be turned into a SWORD module and made available through one of the SWORD repositories.
List of Tamil Bible translations available through ebible.org repository. You should be able to find these after selecting the Tamil language in the module install assistant.
![image](https://github.com/user-attachments/assets/47e2b02a-d34d-4296-81a3-fda019acaee5)
1. *Add support to import sermon text files*. This feature request does not align with the goals of Ezra Bible App. The goal is not primarily to make existing materials available, but rather to support your own Bible study (through tags and notes). Something that may be done eventually is to support the Books feature of SWORD, which would give access to various historic resources available through the SWORD repositories. I believe Spurgeon, Calvin and Bunyan may be available already. You can check for that in other SWORD based apps like Xiphos or BibleTime.
1. *Add support to Sync sermon text files from website folder link or shared folder link*. See point 2) above.
